data/Dels/Dels missions.txt

```diff
@@ -698,6 +698,225 @@ mission "First Contact: Dels Ufulre"
+			`	The message reads, "Officer <first>, you are required to go to <destination> by <date>. There you will receive further orders."`
+
+			choice
+				`	(Accept)`
+					goto accept
+				`	"I don't care what consequences you give me, I'm not going!"`
+					goto decline
+
+			label decline
+			action
+				"dels officer: bad officer" ++
+			``
+				decline
+
+			
+			label accept
+			branch escort
+				random < 20
+			label patrol
+			action
+				"Dels Officer: Patrol Mission" ++
+			``
+				accept
+
+
+
+			label escort
+			action
+				"Dels Officer: Escort Mission" ++
+			``
+				accept
+
+
+	on fail
+		"dels officer: bad officer" ++
+	on abort
+		"dels officer: bad officer" ++
+	on complete
+		"dels officer: report mission available" = 1
+		"Dels Officer: report complete" += 1
+		#Cap the factor to 25 to avoid mission not offering again if done for 30 times.
+		#So minimal chance of offering is 5%
+		"Dels Officer: report complete" <?= 25
+
+#mission "Dels Officer: Participate in defense"
+#	landing
+#	repeat
+#	priority
+#	name `Drive off the Ufulre`
+#	source Delro
+#	destination
+#		government "Dels Kaote" Dels
+#		near Alkurud 0 2
+#	description `There is an Ufulre attack on <destination>. You must participate in driving it off.`
+#	to offer
+#		has "dels officer: report mission available"
+#	on offer
+#		clear "dels officer: report mission available"
+#		conversation
+#			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <first>, you are requested for defense at <destination>. There is an Ufulre attack there."`
+#			`	You prepare to launch.`
+#				accept
+#	on abort
+#		"reputation: Dels Kaote" -= 1
+#		fine 10000
+#		dialog `Officer <first>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
+#
+#	npc evade
+#		government "Dels Ufulre"
+#		personality
+#			confusion 30
+#			plunders harvests target # no staying so they can flee
+#		fleet "Dels Ufulre Raid" 2
+#		system destination
+#
+#	npc
+#		government "Dels Kaote"
+#		personaility
+#			confusion 10
+#			heroic opportunistic frugal staying
+#		fleet "Dels Kaote"
+#		system destination
+#	on complete
+#		payment 100000
+#		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`
+
+mission "Dels Officer: Patrol Systems"
+	landing
+	repeat
+	priority
+	name `Patrol systems`
+	source Delro
+	waypoint
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	waypoint
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	waypoint
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	description `You must patrol <stopovers> to check for Ufulre activity then return to <origin> to report.`
+
+	to offer
+		has "dels officer: report mission available"
+		has "Dels Officer: Patrol Mission" # In testing, I had a stack of these but it only offered once.

```

Comment:
I'm starting a conversation here so we can figure out how to get this to offer more.

Reply:
I have added the `not "Dels Officer: <other__type> Mission"` with this: https://github.com/lumbar527/Galactic-War/commit/41b6f1c2c1c9c44cdbbb348c2e676e6ead6fa586

Reply:
- [x] Works!